client/src/app/employee/employee-create/employee-create.component.html

```diff
@@ -1,82 +1,82 @@
-            aria-label="Users Phone Number"
-          />
-        </div>
-        <div class="mb-1 col-3">
-          <select class="custom-select" formControlName="department">
-            <option value="" disabled>Choose a department</option>
-            <option
-              *ngFor="let department of departments"
-              [ngValue]="department"
-              >{{ department }}</option
-            >
-          </select>
-        </div>
-        <div class="mb-1 col-3">
-          <input
-            type="text"
-            formControlName="title"
-            class="form-control"
-            placeholder="Title"
-            aria-label="Users Title"
-          />
-        </div>
-        <div class="mb-1 col-3">
-          <input
-            type="date"
-            formControlName="employmentStartedOn"
-            class="form-control"
-            placeholder="04/26/2020"
-            aria-label="Users Start Date Of Employment"
-          />
-        </div>
+      <div class="mb-1 col-3">
+        <input
+          type="text"
+          formControlName="lastName"
+          class="form-control"
+          placeholder="Last Name"
+          aria-label="Users Last Name"
+        />
       </div>
-      <div class="row">
-        <div class="mb-1 col-4">
-          <button class="btn btn-outline-info table-margin" (click)="create()"
-            >Create</button
+    </div>
+    <div class="row">
+      <div class="mb-1 col-3">
+        <input
+          type="email"
+          formControlName="email"
+          class="form-control"
+          placeholder="User Email"
+          aria-label="Users Email Address"
+        />
+      </div>
+      <div class="mb-1 col-3">
+        <input
+          type="text"
+          formControlName="phone"
+          class="form-control"
+          placeholder="Phone"
+          aria-label="Users Phone Number"
+        />
+      </div>
+    </div>
+    <div class="row">
+      <div class="mb-1 col-3">
+        <select class="custom-select" formControlName="department">
+          <option value="" disabled>Choose a department</option>
+          <option
+            *ngFor="let department of departments"
+            [ngValue]="department"
+            >{{ department }}</option
           >
-        </div>
+        </select>
+      </div>
+      <div class="mb-1 col-3">
+        <input
+          type="text"
+          formControlName="title"
+          class="form-control"
+          placeholder="Title"
+          aria-label="Users Title"
+        />
+      </div>
+    </div>
+    <div class="row">
+      <div class="mb-1 col-3">
+        <input
+          type="date"
+          formControlName="employmentStartedOn"
+          class="form-control"
+          placeholder="04/26/2020"
+          aria-label="Users Start Date Of Employment"
+        />
+      </div>
+      <div class="mb-1 col-4">
+        <button class="btn btn-outline-info table-margin" (click)="create()"
```

Comment:
To align with HTML form semantics, you can move this event handler to the `(submit)` event of the `<form>`. Then change this button to `<button type="submit" ...>`, or leave off the `type="submit"` because that's the default for `<button>`.

Reply:
I'm going to to use `type="submit"` strictly as a note to myself that the button is "submitting" as it's action. Good to know that is the default behavior of `<button>` though.